src/component/SettingsForm/index.jsx

```diff
@@ -48,10 +50,29 @@ const SettingsForm = () => {
           </div>
         </div>
       ))}
-      <div className=" d-flex flex-row justify-content-center">
-        <Button variant="outline-dark" type="submit" className="w-25 mt-5">Save</Button>
+
+      <h5 className="mt-4 d-flex flex-row justify-content-center">Start Automaticaly Next period</h5>
+      <div className="mt-2 d-flex flex-row justify-content-center">
+        <BootstrapSwitchButton
+          onlabel="ON"
+          offlabel="OFF"
+          onstyle="outline-info"
+          offstyle="outline-dark"
+          width={100}
+          height={30}
+          checked={false}
+          onChange={handelStartAuto}
+        />
+      </div>
+      <div className="d-flex flex-row justify-content-end">
+        <Button variant="outline-dark" type="submit" className="w-23 mt-5">Save</Button>
       </div>
     </Form>
   );
 };
+
+SettingsForm.propTypes = {
+  handelStartAuto: PropTypes.func.isRequired,
```

Comment:
You should rename handelStartAuto -> onShouldAutoStartChange

The naming convention for React props of type function is to prefix them
with `on`, such as `onClick`, `onChange`, `onBlur`, etc. Therefore,
it's better to name our callback function `onShouldAutoStartChange`
because it means "when the `shouldAutoStart` variable value changes, call
the function `toggleShouldAutoStart`."

This comment applies to `Header`, `SettingsModal`, and `SettingsForm` prop `handelStartAuto`.